docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -287,6 +287,14 @@ For read replica clusters, set the value to `10` in all `yb-tserver` and `yb-mas
 
 Default: `6`
 
+##### --leader_lease_duration_ms
+
+The leader lease duration, in milliseconds. A leader keeps establishing a new lease or extending the existing one with every consensus update. A new server is not allowed to serve as a leader (that is, serve up-to-date read requests or acknowledge write requests) until a lease of this duration has definitely expired on the old leader's side.
```

Comment:
> A new server is not allowed to serve as a leader (that is, serve up-to-date read requests or acknowledge write requests) until a lease of this duration has definitely expired on the old leader's side.

this isn't completely accurate, as we have a mechanism whereby the leader can relinquish it's lease. so i might write this:

> A new server is not allowed to serve as a leader (that is, serve up-to-date read requests or acknowledge write requests) until a lease of this duration has definitely expired on the old leader's side, or the old leader has explicitly acknowledged the new leader's lease.

though this also brushes over some details. so maybe a third option is:

> This lease allows the leader to safely serve reads for the duration of its lease, even during a network partition. See: https://docs.yugabyte.com/preview/architecture/transactions/single-row-transactions/#leader-leases-reading-the-latest-data-in-case-of-a-network-partition